drainpipe-dev/scaffold/.twig-cs-fixer.dist.php

```diff
@@ -32,6 +32,9 @@
 // default standard.
 $ruleset->addStandard(new TwigCsFixer\Standard\TwigCsFixer());
 
+// Remove the IncludeFunctionRule which breaks TwigJS.
+$ruleset->removeRule(TwigCsFixer\Rules\Function\IncludeFunctionRule::class);
```

Comment:
Is it possible to do this by committing `.twig-cs-fixer.php` to your local repo? Then you could control this and anything else (since this twigjs isn't commonly used).

Reply:
Agreed. This would be a good thing to just document over changing drainpipe... that is, assuming drainpipe doesn't overwrite files that are present, or the scaffolding can be controlled in composer.